lib/libvcc/vcc_vmod.c

```diff
@@ -106,6 +99,21 @@ vcc_ParseImport(struct vcc *tl)
 	}
 
 	SkipToken(tl, ';');
+	if (osym != NULL) {
+		AN(osym->path);
+		if (strcmp(osym->path, fn) != 0) {
+			VSB_printf(tl->sb, "Warning: Module %.*s imported from different location.\n",
+			    PF(mod));
+			vcc_ErrWhere2(tl, t1, tl->t);
+			VSB_printf(tl->sb, "Previous import was here:\n");
+			vcc_ErrWhere2(tl, osym->def_b, osym->def_e);
+			tl->err = 0; // reset error; this is just a warning
```

Comment:
> A better solution then is to run Varnish with a different search path for vmods when testing.

That doesn't work either if you are using std or depend on any VMOD already installed.